src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -333,8 +333,11 @@ export class TabsTitleControl extends TitleControl {
 				// color, the two colors would collide and the tab border never shows up.
 				// see https://github.com/Microsoft/vscode/issues/33111
 				const activeTabBorderColor = this.getColor(isGroupActive ? TAB_ACTIVE_BORDER : TAB_UNFOCUSED_ACTIVE_BORDER);
+				const activeTabBorderColorTop = this.getColor(isGroupActive ? TAB_ACTIVE_BORDER_TOP : TAB_UNFOCUSED_ACTIVE_BORDER);
```

Comment:
The logic here seems weird. I think you should not fallback to `TAB_UNFOCUSED_ACTIVE_BORDER` because if that color was set, in the bottom if-else case the color is still not applied. I would suggest to introduce a new color `TAB_UNFOCUSED_ACTIVE_BORDER_TOP` to do this properly.

Reply:
Done. Thanks again for the feedback.